src/components/Career/JobCard.js

```diff
@@ -0,0 +1,171 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import JobCardModal from './JobCardModal';
+
+const JobCardWrapper = styled.div`
+  position: relative;
+  width: 100%;
+  height: 567px;
+  display: flex;
+  flex-direction: column;
+  background-color: #181d38;
+  overflow: hidden;
+  padding-top: 29px;
+  border-radius: 8px;
+
+  .header {
+    text-align: center;
+  }
+
+  img {
+    width: 50px;
+  }
+
+  .card-title {
+    margin-top: 30px;
+    color: #fff;
+    font-size: 24px;
+    line-height: 28px;
+    padding: 0 24px 24px;
+    border-bottom: 1px solid #262b48;
+  }
+
+  .content {
+    padding: 24px;
+  }
+
+  .section-title {
+    font-size: 16px;
+    line-height: 19px;
+    color: #ffffff;
+    margin-bottom: 16px;
+  }
+
+  ul {
+    padding: 0;
+    font-size: 14px;
+    line-height: 16px;
+    color: #a1a1a1;
+    li {
+      margin-bottom: 12px;
+    }
+  }
+
+  .responsibilities {
+    margin-top: 24px;
+  }
+
+  .footer {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    background-color: #181d38;
+    color: #a1a1a1;
+    display: flex;
+    padding: 24px;
+
+    .view-more {
+      background: #262b48;
+      margin-left: 16px;
+    }
+
+    .apply-job {
+      background: #ebbf6e;
+    }
+
+    .footer-button {
+      flex: 1;
+      border-radius: 8px;
+      line-height: 16px;
+      font-size: 14px;
+      text-align: center;
+      color: #fff;
+      cursor: pointer;
+      padding: 9px 16px;
+    }
+
+    @media (max-width: 1200px) {
+      flex-wrap: wrap;
+    }
+  }
+`;
+
+// send mails
+function onClickApplyButton() {
+  window.location.href = 'mailto:career@venus.io';
```

Comment:
Can we add a predefined subject line for each of the positions? Sth like:

```suggestion
function onClickApplyButton(subject) {
  window.location.href = `mailto:career@venus.io?subject=${encodeURIComponent(subject)}`;
```

Not strictly a requirement (I don't think a lot of emails would come, so there won't be a mess), but nice to have to simplify our HRs work a bit, and to free the candidates of writing the subject by themselves.